frontend/components/FightInfo.js

```diff
@@ -0,0 +1,116 @@
+    }, []);
+
+    return (
+        <>
+            <FormHeader name="Fight Details"></FormHeader>
+            <Center>
+                {/* Fight date-time */}
+                <Flex direction="row" >
+                    <FontAwesome name="calendar" size={30} color="black" />
+                    <Text marginLeft={5} color="black" fontSize={22} fontWeight="bold" >
+                        {fightData.date} {fightData.time}
+                    </Text>
+                </Flex>
+
+                {/* Fighter1 vs Fighter2 */}
+                <Flex direction="row" marginTop="20px" rounded="lg" borderColor="black" borderWidth="1" p="4px">
+                    <FontAwesome name="user-circle" size={30} color="black" />
+                    <Text marginLeft={5} color="black" fontSize={22} fontWeight="bold" >
+                        {fighter1.firstName}  {fighter1.lastName}
+                    </Text>
+                </Flex>
+                <Text marginTop="10px" marginBottom="10px" color="black" fontSize={22} fontWeight="bold" >VS</Text>
+                <Flex direction="row" rounded="lg" borderColor="black" borderWidth="1" p="4px">
+                    <FontAwesome name="user-circle" size={30} color="black" />
+                    <Text marginLeft={5} color="black" fontSize={22} fontWeight="bold" >
+                        {fighter2.firstName}  {fighter2.lastName}
+                    </Text>
+                </Flex>
+
+                {/* Rules, TODO: api/matches/id/{id} seems to not contain rules info */}
+                <Text marginTop="30px" color="black" fontSize={22} fontWeight="bold" >Rules:</Text>
+                <Text marginLeft="40px" marginRight="40px" color="grey" fontSize={16}>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the </Text>
+
+                {/* Joining codes for referees */}
+                <Text marginTop="20px" color="black" fontSize={22} fontWeight="bold" >Join codes:</Text>
+                <VStack alignItems="center">
+                    <Text marginTop="10px" color="black" fontSize={18} fontWeight="bold">Main Referee:</Text>
+                    <Text color="black" fontSize={18} >{fightData.mainRefereeToken}</Text>
+                    <Text marginTop="10px" color="black" fontSize={18} fontWeight="bold">Side Referee 1:</Text>
+                    <Text color="black" fontSize={18} >{fightData.sideRefereeToken1}</Text>
+                    <Text marginTop="10px" color="black" fontSize={18} fontWeight="bold">Side Referee 2:</Text>
+                    <Text color="black" fontSize={18} >{fightData.sideRefereeToken2}</Text>
+                </VStack>
+
+                {/* Fight cancelling button TODO: send request to backend */}
+                <Button marginTop="40px" colorScheme="red" 
+                    onPress={() => console.log("Handling fight cancelling...")}>Cancel fight</Button>
+            </Center>
```

Comment:
```suggestion
            <ScrollView>
            <Center>
                {/* Fight date-time */}
                <Flex direction="row" >
                    <FontAwesome name="calendar" size={30} color="black" />
                    <Text marginLeft={5} color="black" fontSize={22} fontWeight="bold" >
                        {fightData.date} {fightData.time}
                    </Text>
                </Flex>

                {/* Fighter1 vs Fighter2 */}
                <Flex direction="row" marginTop="20px" rounded="lg" borderColor="black" borderWidth="1" p="4px">
                    <FontAwesome name="user-circle" size={30} color="black" />
                    <Text marginLeft={5} color="black" fontSize={22} fontWeight="bold" >
                        {fighter1.firstName}  {fighter1.lastName}
                    </Text>
                </Flex>
                <Text marginTop="10px" marginBottom="10px" color="black" fontSize={22} fontWeight="bold" >VS</Text>
                <Flex direction="row" rounded="lg" borderColor="black" borderWidth="1" p="4px">
                    <FontAwesome name="user-circle" size={30} color="black" />
                    <Text marginLeft={5} color="black" fontSize={22} fontWeight="bold" >
                        {fighter2.firstName}  {fighter2.lastName}
                    </Text>
                </Flex>

                {/* Rules, TODO: api/matches/id/{id} seems to not contain rules info */}
                <Text marginTop="30px" color="black" fontSize={22} fontWeight="bold" >Rules:</Text>
                <Text marginLeft="40px" marginRight="40px" color="grey" fontSize={16}>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the </Text>

                {/* Joining codes for referees */}
                <Text marginTop="20px" color="black" fontSize={22} fontWeight="bold" >Join codes:</Text>
                <VStack alignItems="center">
                    <Text marginTop="10px" color="black" fontSize={18} fontWeight="bold">Main Referee:</Text>
                    <Text color="black" fontSize={18} >{fightData.mainRefereeToken}</Text>
                    <Text marginTop="10px" color="black" fontSize={18} fontWeight="bold">Side Referee 1:</Text>
                    <Text color="black" fontSize={18} >{fightData.sideRefereeToken1}</Text>
                    <Text marginTop="10px" color="black" fontSize={18} fontWeight="bold">Side Referee 2:</Text>
                    <Text color="black" fontSize={18} >{fightData.sideRefereeToken2}</Text>
                </VStack>

                {/* Fight cancelling button TODO: send request to backend */}
                <Button marginTop="40px" colorScheme="red" 
                    onPress={() => console.log("Handling fight cancelling...")}>Cancel fight</Button>
            </Center>
            </ScrollView>
```

Reply:
Done, wrapped with scroll view!